webdriver/webdriverio/package.json

```diff
@@ -0,0 +1,18 @@
+{
+    "name": "webdriverio",
+    "version": "1.0.0",
+    "private": true,
+    "scripts": {
+        "test": "wdio run wdio.conf.js"
+    },
+    "dependencies": {
+        "@wdio/cli": "^7.34.0"
+    },
```

Comment:
This should be a test dependency. It is not needed during runtime. Please remove and re-add it using `npm install --test`.